app/controllers/report_controller/reports/editor.rb

```diff
@@ -628,6 +628,10 @@ def gfv_performance
     end
   end
 
+  def chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@lpichler Id love to but brakeman fails this type of action because it constantizes a string given by the user